webrtc-stats.html

```diff
@@ -2989,6 +2989,17 @@ <h2>
                   addresses, and fully qualified domain names (FQDNs). See
                   [[!RFC5245]] section 15.1 for details.
                 </p>
+                <div class="note">
+                  A user agent should make sure to limit exposure of remote candidate addresses to
+                  the ones already known to the web application, in particular peer reflexive remote
+                  candidate addresses. If a user agent cannot tell whether a remote candidate
+                  address is known to the web application, it MUST NOT surface this address value in
+                  any RTCICECandidateStats dictionary. Once a RTCPeerConnection instance is notified
+                  by the web application of an address using <a
+                  href="https://w3c.github.io/webrtc-pc/#dom-rtcpeerconnection-addicecandidate">addIceCandidate</a>
+                  the user agent can surface the 'address' member in any remote RTCIceCandidateStats
+                  dictionary of the corresponding RTCPeerConnection.
```

Comment:
I think we need to say something other than "MUST NOT surface".
The choices I see are:
- Omit the whole stat (which would hurt overall clarity of stats)
- Leave out the address string
- Generate a name in the same way we generate MDNS names, and enter that instead, possibly with a prefix that allows identification of "hidden" rather than "mdns".
A quick scan finds an IP address only in the candidate stats - offhand, I'd prefer alternative 3.

Reply:
The idea was to not have any 'address' member in RTCICECandidateStats in that case since this is not a required field. I'll clarify the PR.
I can go with alternative 3 but I do not really see the value.
Is the concern that scripts might break if this field does not show up?

Reply:
If it is known that the address is peer-reflexive, I would go with option 3. Although would that not require a reserved value for hidden? 

However, if we go with option 2, I am assuming omit means null, I want to make sure that we set it to null only when it needs to be hidden. Although it might be set to null when the remote address is unknown?

Reply:
Omit means 'undefined'.
I am fine with an explicit 'hidden' if that helps but I am wondering whether that is worth it.
If we want to make it explicit, I would go with null instead of undefined, which would probably require changing the IDL.

Reply:
JS makes a difference between "undefined" and "null". We should have "null" here.